core/templates/pages/collection-editor-page/services/collection-editor-state.service.ts

```diff
@@ -72,13 +72,12 @@ export class CollectionEditorStateService {
    * specified collection ID. See setCollection() for more information on
    * additional behavior of this function.
    */
-  loadCollection(collectionId: string, successCallback: () => void): void {
+  loadCollection(collectionId: string): void {
```

Comment:
Why did we take out the `successCallback`?

Reply:
`successCallback` is not used now anywhere in the codebase.